packages/router/src/splash-page.tsx

```diff
@@ -1,134 +1,538 @@
-const SplashPage: React.VFC = () => (
-  <main>
-    <section>
-      <style
-        dangerouslySetInnerHTML={{
-          __html: `
-            html, body {
+import React, { useState, useEffect } from 'react'
+
+interface SplashPageProps {
+  hasGeneratedRoutes: boolean
+  routes: any[]
```

Comment:
Is there already an Interface for routes? I'm passing in an array of routes here and just using the type `any []` because I couldn't find a type for routes

Reply:
Hey @alicelovescake - there is indeed a routes interface - you can import `AvailableRoutes` from `@redwoodjs/router`.

This particular interface gets updated dynamically (using type generation) based on the routes you have in the project.

Not sure if it makes sense to use here, but will leave that to your discretion.